etc/kayobe/ansible/deploy-os-capacity-exporter.yml

```diff
@@ -3,6 +3,13 @@
   gather_facts: false
 
   tasks:
+    - name: Ensure legacy os_exporter.cfg config file is deleted
+      ansible.builtin.file:
+        path: /etc/kolla/haproxy/services.d/os_exporter.cfg
+        state: absent
+      delegate_to: controllers
```

Comment:
This should be delegated to the hosts running HAProxy, which are network nodes by default.

Reply:
Changed to `network`